core/src/components/slider/slider.ts

```diff
@@ -624,6 +629,32 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 		api: {},
 		directives: {
 			sliderDirective: mergeDirectives(sliderDirective, resizeDirective),
+			handleDirective: createAttributesDirective((handleContext$: ReadableSignal<{item: SliderHandle}>) => ({
+				events: {
+					keydown: (event: KeyboardEvent) => widget.actions.keydown(event, handleContext$().item.id),
+					mousedown: (event: MouseEvent) => widget.actions.mouseDown(event, handleContext$().item.id),
+					touchstart: (event: TouchEvent) => widget.actions.touchStart(event, handleContext$().item.id),
+				},
+				attributes: {
+					role: 'slider',
+					'aria-valuemin': min$(),
+					'aria-valuemax': max$(),
+					'aria-valuenow': handleContext$().item.value,
+					'aria-valuetext': handleContext$().item.ariaValueText,
+					'aria-label': handleContext$().item.ariaLabel,
+					'aria-orientation': vertical$() ? 'vertical' : undefined,
+					'aria-disabled': disabled$() ? 'true' : undefined,
+					disabled: disabled$() ? 'true' : undefined,
+					'aria-readonly': readonly$() ? 'true' : undefined,
+				},
+				styles: {
+					left: computed(() => handleDisplayOptions$()[handleContext$().item.id].left + '%'),
+					top: computed(() => handleDisplayOptions$()[handleContext$().item.id].top + '%'),
+				},
+				classNames: {
+					//TODO: add classes once directives can be customized.
```

Comment:
I couldn't put `classNames` because otherwise they will be there even if in the custom handles I want them to be replaced by custom classes. It is an example of how important is the customization option.

Reply:
It's true that it's going to add classes, but I would say that it's not a problem as here, it's just specific classes from AgnosUI, not bootstrap. So I think it's fine to manage them from here, it's just classes that identify this specific element. We can have the same question for the other attributes.

But you're right, we'll need a way to remove them if a dev absolutely want to do so, and it will be part of the customization, that will be implemented later (the classes will have the same possibility of customization as the event, attributes an styles).

So, to be discussed, but my thought is that you can add them (only bootstrap specific must be avoided).

Reply:
Ok, I only had to override bootstrap `_slider.scss` classes to get the custom styles I wanted.
A user of headless component won't have any css included by default, so it will be up to him to add css to the proper classes added by default by our API